src/components/ebay-table/marko-tag.json

```diff
@@ -0,0 +1,48 @@
+{
+    "attribute-groups": ["html-attributes"],
+    "@*": {
+        "type": "expression",
+        "targetProperty": null
+    },
+    "@html-attributes": "expression",
+    "@density": {
+        "enum": ["compact", "relaxed"]
+    },
+    "@column <column>[]": {
```

Comment:
@agliga Discussed with @LuLaValva and using `column` instead of `columns` makes it more compatible with Marko5/6 where we don't use `marko-tag.json`. Let me know your thoughts.